src/main/kotlin/no/nav/syfo/util/DateUtil.kt

```diff
@@ -1,13 +1,14 @@
 package no.nav.syfo.util
 
 import java.sql.Timestamp
-import java.time.OffsetDateTime
-import java.time.ZoneOffset
+import java.time.*
 
 val defaultZoneOffset: ZoneOffset = ZoneOffset.UTC
 
 val nowTimestampUTC: Timestamp = Timestamp.from(OffsetDateTime.now(defaultZoneOffset).toInstant())
 
 fun Timestamp.toOffsetDateTimeUTC(): OffsetDateTime = this.toInstant().atOffset(defaultZoneOffset)
 
-fun Timestamp?.toOffsetDateTimeUTCOrNull(): OffsetDateTime? = this?.toInstant()?.atOffset(defaultZoneOffset)
+fun OffsetDateTime.toLocalDateTimeOslo() = this.toInstant()?.atOffset(
+    ZoneId.of("Europe/Oslo").rules.getOffset(Instant.now())
+)?.toLocalDateTime()
```

Comment:
Vet det hva dette jeg foreslo, men det er en potensiell bug'er siden Instant.now() brukes til å hente offset (dvs sommertid eller ikke). Så hvis det feks er et tidspunkt i februar som en bruker skal se på i juli så blir det feil.

Googlet litt: Tror dette er den beste måten å gjøre om en OffsetDateTime til en LocalDateTime: 
this.atZoneSameInstant(ZoneId.of("Europe/Oslo")).toLocalDateTime()

Reply:
Endret til det nå.